src/main/resources/defaultProperties/JiraXRay.properties

```diff
@@ -0,0 +1,22 @@
+########################################################
```

Comment:
next step is to add this to the online configuration manager, I think we can merge it with another properties file... let's have a meeting to discuss.